commands/print_env.go

```diff
@@ -66,11 +72,9 @@ func NewPrintEnv(
 }
 
 func (p PrintEnv) CheckFastFails(subcommandFlags []string, state storage.State) error {
```

Comment:
@rowanjacobs If this function always returns `nil`, should it just be removed?

Reply:
`CheckFastFails` is unfortunately part of the interface for commands, so it can't be removed

Reply:
Ah, I see. Fair enough